Makefile

```
@@ -0,0 +1,116 @@
+# -- ARNOLD
+ARNOLD_IMAGE_NAME      = arnold
+ARNOLD_IMAGE_TAG       = $(shell tr -d '\n' < VERSION)
+ARNOLD_IMAGE_TAG_DEV   = $(ARNOLD_IMAGE_TAG)-dev
+ARNOLD_IMAGE           = $(ARNOLD_IMAGE_NAME):$(ARNOLD_IMAGE_TAG)
+ARNOLD_IMAGE_DEV       = $(ARNOLD_IMAGE_NAME):$(ARNOLD_IMAGE_TAG_DEV)
+ANSIBLE_VAULT_PASSWORD = arnold
```

Should it use environment variables instead of these values, when defined ?

```suggestion
ARNOLD_IMAGE_NAME      ?= arnold
ARNOLD_IMAGE_TAG       ?= $(shell tr -d '\n' < VERSION)
ARNOLD_IMAGE_TAG_DEV   ?= $(ARNOLD_IMAGE_TAG)-dev
ARNOLD_IMAGE           ?= $(ARNOLD_IMAGE_NAME):$(ARNOLD_IMAGE_TAG)
ARNOLD_IMAGE_DEV       ?= $(ARNOLD_IMAGE_NAME):$(ARNOLD_IMAGE_TAG_DEV)
ANSIBLE_VAULT_PASSWORD ?= arnold
```

---

This is what I did in a first attempt, and, I changed my mind to enforce default development values. Do you have a particular scenario in mind where it could be useful to redefine those thanks to the environment?

---

One thing that comes to my mind is that CircleCI is using `make` in its jobs. And also, it defines these some of these environment variables in `ci_env` task. At this time, the values of these environment variables are the same than the default values defined in the Makefile.

But if someone wants to change the variables defined in `ci_env` job in the future, I'm pretty sure that this will lead to unexpected behavior and confusion.

---

Ok then.